cmd/buildah/manifest.go

```diff
@@ -0,0 +1,653 @@
+package main
```

Comment:
It might save some work when integrating the functionality into podman, but it'd also need to duplicate the `getStore()`, `getContext()` helper functions that we keep in `common.go`, or make them replaceable so that libpod can tie into its runtime structures and startup/shutdown logic.  At that point I don't really know if it ends up being less work than having it wrap its own CLI around the `manifests` package.